tests/unit_tests/test_wave_01.py

```diff
@@ -49,7 +49,7 @@ def test_removing_not_found_is_false():
 
     result = vendor.remove(item)
 
-    raise Exception("Complete this test according to comments below.")
+    # raise Exception("Complete this test according to comments below.")
```

Comment:
❗ Besides commenting out this Exception, you need to write assertions for this test too